tests/e2e/gravity-forms.spec.js

```diff
@@ -46,12 +64,31 @@ test('check the Gravity Forms confirmation message, text type', async ({requestU
   await page.goto('./wp-admin/admin.php?page=gf_entries');
   const latestEntry = page.locator('#the-list > tr.entry_row').first();
   await expect(latestEntry).toBeVisible();
-  await expect(latestEntry.locator('td[data-colname="Name"]')).toContainText(TEST_NAME);
+  await expect(latestEntry.locator('td[data-colname="First name"]')).toContainText(TEST_FIRST_NAME);
+  await expect(latestEntry.locator('td[data-colname="Last name"]')).toContainText(TEST_LAST_NAME);
   await expect(latestEntry.locator('td[data-colname="Email"]')).toContainText(TEST_EMAIL);
+});
 
-  // Delete the form.
-  await page.request.delete(`./wp-json/gf/v2/forms/${createdForm.id}`);
+test('check the Gravity Forms Hubspot feeds', async ({page}) => {
+  // Add a Hubspot feed.
+  await page.goto(`./wp-admin/admin.php?page=gf_edit_forms&view=settings&subview=gravityformshubspot&id=${createdForm.id}`);
+  const createFeedLink = page.getByRole('link', {name: 'create one'});
+  // If the Huspot add-on isn't activated, the page will be empty so this link won't be there.
+  if (await createFeedLink.isVisible()) {
+    await createFeedLink.click();
+    // Set lead status to "new".
+    await page.selectOption('#_hs_customer_hs_lead_status', {label: 'New'});
+    // Map contact fields.
+    await page.selectOption('#_hs_customer_firstname', {label: 'First name'});
+    await page.selectOption('#_hs_customer_lastname', {label: 'Last name'});
+    await page.selectOption('#_hs_customer_email', {label: 'Email'});
+    // Save Hubspot feed.
+    await page.getByRole('button', {name: 'Save Settings'}).click();
+    await expect(page.locator('.gforms_note_success')).toBeVisible();
 
-  // Disable Gravity Forms rest API.
-  await toggleRestAPI({page}, false);
+    // Check that the feed is active.
+    await page.goto(`./wp-admin/admin.php?page=gf_edit_forms&view=settings&subview=gravityformshubspot&id=${createdForm.id}`);
+    const hubspotFeed = page.locator('#the-list > tr').first();
+    await expect(hubspotFeed.locator('gform-status-indicator-status')).toContainText('Active');
```

Comment:
It works for me if I add a dot for the locator `locator('.gform-status-indicator-status')`, but fails otherwise :thinking:

Reply:
That makes sense 🤦‍♀️ weird that it works for me and on the CI